loadtest/store/memstore/random.go

```diff
@@ -12,8 +12,9 @@ import (
 )
 
 var (
-	ErrEmptyMap   = errors.New("memstore: cannot select from an empty map")
-	ErrEmptySlice = errors.New("memstore: cannot select from an empty slice")
+	ErrEmptyMap     = errors.New("memstore: cannot select from an empty map")
+	ErrEmptySlice   = errors.New("memstore: cannot select from an empty slice")
+	ErrSizeMismatch = errors.New("memstore: cannot select from a slice, not enough elements")
```

Comment:
Detail: I come from a [world](https://en.wikipedia.org/wiki/Sizeof) where `size` and `length` are very different concepts. I propose we change this since what we are really checking is the length. Also `s.users` is a `map` and not a `slice`.